README.md

```diff
@@ -136,11 +136,36 @@ The unique characteristics of cases of this class are as follows:
 |Small cell cancer of the lung, somatic, 182280 (3)|RB1|614041|13q14.2|
 |Small-cell cancer of lung (2)|SCLC1|182280|3p23-p21|
 
-
 **All known cases**:
 There is a spreadsheet which collates all known cases as of 2024/11/18: [google sheet](
 https://docs.google.com/spreadsheets/d/1hKSp2dyKye6y_20NK2HwLsaKNzWfGCMJMP52lKrkHtU/). The MIMs of the known cases are: `159595`, `182280`, `607107`, and `615830`.
 
+**Additional notes**:
+Note that unlike the other `review.tsv` cases, for this one, a single case spans multiple rows. The cases can be 
+identified easily because they are enumerated with a leading number in the `value` column, e.g. "1: " for the first case
```

Comment:
where is this leading number found related to `leading number in the `value` column, e.g. "1: "`?

Reply:
It feels a little clunky to describe it, let me know if you come up with any better phrasing.

It is exactly where this tries to describe it: It's the leading number in the `value` column. You can observe in the [google sheet](https://docs.google.com/spreadsheets/d/1R4yKwiwkAWoWYObh415kyilRpRV4UnbAe1Oq4WBxkfI/edit?gid=974592092#gid=974592092). Look at rows 9 through 15, which are of type "D2G: self-referential".

Reply:
Oh, I didn't realize you were referring to how this case is presented/displayed in the `review.tsv` file itself.

Reply:
Hmm, actually it is ambiguous. "Multiple rows" _could_ refer to a different file; `morbidmap.txt`. Indeed there are multiple rows in that file.

I'll update this sentence to be more clear!